types/wordpress__components/card/index.d.ts

```diff
@@ -7,14 +7,27 @@ declare namespace Card {
         className?: string | undefined;
 
         /**
-         * Renders a borderless card.
+         * Size of the elevation shadow, based on the Style system’s elevation
+         * system. This may be helpful in highlighting certain content. For more
+         * information, check out [Elevation]{@link https://developer.wordpress.org/block-editor/designers-developers/developers/components/elevation/}.
+         *
+         * @defaultValue 0
+         */
+        elevation?: number | undefined;
+
+        /**
+         * Renders without a border.
+         *
+         * @defaultValue false
          */
         isBorderless?: boolean | undefined;
 
         /**
-         * Renders an elevated card.
+         * Renders with rounded corners.
+         *
+         * @defaultValue false
          */
-        isElevated?: boolean | undefined;
```

Comment:
How do you feel about keeping `isElevated` but marking it as `@deprecated`?

Reply:
Can do! The only reason I removed this one instead of marking it deprecated is the prop was actually removed upstream, rather than being deprecated but still properly implemented (as are the Button variants, for example).

Reply:
Unless I'm looking in the wrong place, upstream Card still supports `isElevated`: https://github.com/WordPress/gutenberg/blob/6eff9a514591f3b14f667e94b745eaaa507d912a/packages/components/src/card/card/hook.js#L25

Reply:
You are absolutely correct, I must've been looking in the wrong place. I'll add it in as deprecated!